src/ConfigWidgets/SymbolsDialogTest.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+#include <qtestmouse.h>
+
+#include <QDialogButtonBox>
+#include <QListWidget>
+#include <QPushButton>
+#include <QTest>
+#include <filesystem>
+#include <vector>
+
+#include "ConfigWidgets/SymbolsDialog.h"
+
+namespace orbit_config_widgets {
+
+TEST(SymbolsDialog, GetSetSymbolPaths) {
+  SymbolsDialog dialog;
+  auto* list_widget = dialog.findChild<QListWidget*>("listWidget");
+
+  // dialog.show();
+  EXPECT_EQ(list_widget->count(), 0);
+  EXPECT_EQ(dialog.GetSymbolPaths(), std::vector<std::filesystem::path>{});
+
+  const std::vector<std::filesystem::path> test_paths{"test/path", "test_path2"};
+
+  dialog.SetSymbolPaths(test_paths);
+  EXPECT_EQ(list_widget->count(), test_paths.size());
+  EXPECT_EQ(dialog.GetSymbolPaths(), test_paths);
+}
+
+TEST(SymbolsDialog, AddSymbolPath) {
+  SymbolsDialog dialog;
+  auto* list_widget = dialog.findChild<QListWidget*>("listWidget");
+  EXPECT_EQ(list_widget->count(), 0);
+
+  std::filesystem::path path{"test/path/to/file.txt"};
```

Comment:
Maybe add tests with other path formats, at least Windows since it's our primary platform?

Reply:
In general I think these paths should be absolute. And yes. Windows please. :-)

Reply:
I made them all absolute and added windows paths as well